addons/binding/org.openhab.binding.yamahareceiver/README.md

```diff
@@ -23,6 +23,26 @@ Bridge yamahareceiver:yamahaAV:ReceiverID "Yamaha Receiver Bridge Name" [host="a
 If your receiver is using menu-based net radio navigation, you can use this binding to
 select radio stations from a configured menu.
 
+Configuration parameters for Bridge type `yamahareceiver`:
+
+Parameter | Required | Default | Description
+--------|------|------|------
+`host` | yes | / | The IP address of the AVR to control
+`port` | no | 80 | The API port of the AVR to control
+`refreshInterval` | no | 60 | Refresh interval in seconds
+`albumUrl` | no | / | When the album image is not provided by the Yamaha input source, you can specify the default image URL to apply
```

Comment:
The default value is a rather fancy vinyl image ;) I suggest to maybe do `(embedded image URL)` rather than /

Reply:
Here I was not sure whether you mean I should add the embedded URL or just a hint that an embedded URL is used. I decided to go with option 2 as it also was not clear for me what this embedded URL actually is.